conjur/wrapper/http_wrapper.py

```diff
@@ -29,14 +29,13 @@ class HttpVerb(Enum):
     DELETE = 4
     PATCH = 5
 
-
 # pylint: disable=too-many-locals,consider-using-f-string
 # ssl_verify can accept Boolean or String as per requests docs
 # https://requests.readthedocs.io/en/master/api/#main-interface
 def invoke_endpoint(http_verb: HttpVerb, endpoint: ConjurEndpoint, params: dict, *args,
                     check_errors: bool = True, ssl_verify: bool = True,
                     auth: tuple = None, api_token: str = None,
-                    query: dict = None) -> requests.Response:
+                    query: dict = None, headers: dict = {}) -> requests.Response:
```

Comment:
This is very dangerous [and possibly  the source of all evil ](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)`headers: dict = {}` pycharm should warn you about this.

Reply:
great point @mbenita-Cyberark